clientconn.go

```diff
@@ -904,15 +890,17 @@ func (ac *addrConn) errorf(format string, a ...interface{}) {
 
 // resetTransport recreates a transport to the address for ac.  The old
 // transport will close itself on error or when the clientconn is closed.
+//
+// The connectivity state of ac will be set to TransientFailure if it's not
```

Comment:
Is this accurate? It seems that `ac.state` will be set to `connectivity.Connecting` unconditionally and then to `connectivity. TransientFailure` on every attempt; I can't tell which part of this is sensitive to "if it's not first time doing resetTransport."

Reply:
The state is set by `transportMonitor` to `TransientFailure` before this function is called.
I removed this comment and add a comment in `transportMonitor`.